Correzione adr (#214)
SWEet16-SWE-Group/docs: rtb/analisi-requisiti/src/requisiti.tex

     RFO54 & Il cliente deve poter aprire un canale di comunicazione tramite chat verso un ristoratore & UC49 \\ \hline
    \end{longtblr}
    \end{center}

\subsubsection{Facoltativi}

\begin{center}
    \begin{tblr}{
        colspec={|X[1.5cm]|X[3cm]|X[8cm]|X[2.5cm]|},
        row{odd}={bg=white},
        row{even}={bg=lightgray}
        }
        \hline

        \textbf{Codice} & \textbf{Caratteristica} & \textbf{Descrizione} & \textbf{Fonte} \\

>          RFD3 & Sicurezza & Le comunicazioni tra il client e il server devono essere crittografate, data la natura sensibile dei dati inseriti dagli utenti (SSL/TLS) & Capitolato \\ \hline

Reviewer: ```suggestion
         RFD3 & Sicurezza & Le comunicazioni tra il client e il server devono essere crittografate, data la natura sensibile dei dati inseriti dagli utenti (tramite SSL/TLS) & Capitolato \\ \hline
```

>          RFD3 & Sicurezza & Le comunicazioni tra il client e il server devono essere crittografate, data la natura sensibile dei dati inseriti dagli utenti (tramite SSL/TLS) & Capitolato \\ \hline

Reviewer 2: da mettere SSL/TLS con apice riferito al glossario in cui diciamo di che si tratta?

>          RFF4 & Sicurezza & Viene implementato RSA come sistema di autenticazione per controllare l'accesso ai dati degli utenti & Capitolato \\ \hline

Reviewer: > Per controllare l'accesso ai dati degli utenti

Non mi piace molto come è formulata questa frase.

Sei sicuro che sia corretto chiamare RSA sistema di autenticazione? Alla fine è una crittografia asimmetrica, cosa c'entra questo con l'autenticazione?

>          RFF4 & Sicurezza & Viene implementato un sistema di autenticazione per controllare l'accesso ai dati degli utenti & Capitolato \\ \hline

Reviewer 2: ma possiamo dire che è facoltativo il sistema di autenticazione?

>          RFD5 & Privacy & Viene richiesto all'utente un esplicito permesso per memorizzare i messaggi delle sue chat che saranno comunque salvati in forma crittografata end-to-end & Capitolato \\ \hline

Reviewer 2: ```suggestion
         RFD5 & Privacy & Viene richiesto all'utente un esplicito permesso per memorizzare i messaggi delle sue chat ;nel caso in cui il permesso venga concesso, i messaggi saranno salvati in forma crittografata end-to-end & Capitolato \\ \hline
```

        }
        \hline
        \textbf{Codice} & \textbf{Caratteristica} & \textbf{Descrizione} & \textbf{Fonte} \\

        RNFQO1 & Sviluppo & Il sistema sarà sviluppato seguendo quanto scritto nelle Norme di Progetto & Brainstorming Interno \\ \hline

>         RNFQO2 & Sviluppo & Il progetto sarà pubblicato su GitHub & Brainstorming Interno \\ \hline

Reviewer: ```suggestion
        RNFQO2 & Sviluppo & Il progetto sarà pubblicato su GitHub & Capitolato \\ \hline
```

Author: perché il capitolato ?

>         RNFQD3 & Sviluppo & Il progetto deve essere accompagnato con un manuale utente & Brainstorming Interno \\ \hline

Reviewer: Non sono sicuro serva veramente il manuale utente, io toglierei completamente questa riga, facciamo sempre ora ad aggiungerlo nella versione 3.0.0

        \end{tblr}
\end{center}

\subsubsection{Vincolo}

\begin{center}
    \begin{tblr}{
        colspec={|X[1.5cm]|X[3cm]|X[8cm]|X[2.5cm]|},
        row{odd}={bg=white},
        row{even}={bg=lightgray}
        }
        \hline
        \textbf{Codice} & \textbf{Caratteristica} & \textbf{Descrizione} & \textbf{Fonte} \\

>         RNFVO1 & Sviluppo & Il sitema sarà sviluppato e consegnato tramite piattaforma Docker & Capitolato \\ \hline
>         RNFVO2 & Portabilità & Il sitema dovrà funzionare sul browser Firefox dalla versione più recente (124.0) & Capitolato \\ \hline
>         RNFVO3 & Portabilità & Il sitema dovrà funzionare sul browser Chromium dalla versione più recente (123.0.6312.58) & Capitolato \\ \hline
>         RNFVO4 & Portabilità & Il sitema dovrà funzionare sul browser Safari dalla versione più recente (5.1.7) & Capitolato \\ \hline

Reviewer: ```suggestion
        RNFVO1 & Sviluppo & Il sistema sarà sviluppato e consegnato tramite piattaforma Docker & Capitolato \\ \hline
        RNFVO2 & Portabilità & Il sistema dovrà funzionare sul browser Firefox dalla versione più recente (124.0) & Capitolato \\ \hline
        RNFVO3 & Portabilità & Il sistema dovrà funzionare sul browser Chromium dalla versione più recente (123.0.6312.58) & Capitolato \\ \hline
        RNFVO4 & Portabilità & Il sistema dovrà funzionare sul browser Safari dalla versione più recente (5.1.7) & Capitolato \\ \hline
```

Non è sensato pensare di far funzionare l'applicazione solo nella versione più recente di firefox, chromium o safari. 
Dovrebbe funzionare nella versione di 2/3 anni fa per avere un minimo di compatibilità.

        }
        \hline
        \textbf{Codice} & \textbf{Caratteristica} & \textbf{Descrizione} & \textbf{Fonte} \\

>         RNFQO1 & Sviluppo & Il sistema sarà sviluppato seguendo quanto scritto nelle Norme di Progetto & Brainstorming Interno \\ \hline

Reviewer 2: Non è forse meglio riferirsi al piano di qualifica? Cardin parla di qualità dei processi di sviluppo; forse è meglio citare esplicitamente la soglia della copertura dell'80% di codice nei test

Reviewer: Serve citare esplicitamente la soglia di copertura dei test all'80%

        }
        \hline
        \textbf{Codice} & \textbf{Caratteristica} & \textbf{Descrizione} & \textbf{Fonte} \\

        RQO1 & Sviluppo & Il prodotto deve essere sviluppato in modo concorde a quanto stabilito nelle Norme di Progetto & Brainstorming Interno \\ \hline
        RQO2 & Sviluppo & Il progetto deve essere accessibile pubblicamente su GitHub$^{G}$ & Brainstorming Interno \\ \hline
        RQO3 & Sviluppo & Devono essere realizzati i test di unità ed integrazione con una copertura minima dell'80\% & Capitolato \\ \hline
        RQO4 & Sviluppo & Deve essere fornita una completa documentazione sulle scelte implementative e progettuali e relative motivazioni & Capitolato \\ \hline
        RQO5 & Sviluppo & Deve essere fornita una completa documentazione sugli eventali problemi aperti e sulle possibili soluzioni da esplorare & Capitolato \\ \hline

>         RQD6 & Sviluppo & Deve essere fornita un’analisi rispetto al carico massimo supportato in numero di dispositivi e dell'eventuale servizio cloud più adatto & Capitolato \\ \hline

Author: RQ_6  segnato come desiderabile qui
